apps/antalmanac/src/components/buttons/LoadSchedule.tsx

```diff
@@ -0,0 +1,140 @@
+import { useCallback, useEffect, useState } from 'react';
+import {
+    Button,
+    type ButtonProps,
+    Checkbox,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogContentText,
+    DialogTitle,
+    FormControlLabel,
+    TextField,
+    Tooltip,
+} from '@mui/material';
+import { LoadingButton } from '@mui/lab';
+import { CloudDownload as CloudDownloadIcon } from '@mui/icons-material';
+
+import { loadSchedule } from '$actions/AppStoreActions';
+import { isDarkMode } from '$lib/helpers';
+import { useLoadingStore } from '$stores/loading';
+
+function getSavedUserId(): string | null {
+    if (typeof window === 'undefined') {
+        return null;
+    }
+
+    const userId = window.localStorage.getItem('userID');
+    return userId;
+}
+
+export function LoadScheduleButton(props: ButtonProps = {}) {
+    const [loadedSchedule, setLoadedSchedule] = useLoadingStore((state) => [
+        state.loadedSchedule,
+        state.setLoadedSchedule,
+    ]);
+
+    const [open, setOpen] = useState(false);
+
+    const [rememberMe, setRememberMe] = useState(true);
```

Comment:
TODO is to use a form library that can manage the states of the inputs for us, instead of declaring a new `useState` for every single input.